### DISCUSSION TITLE
Allow unfollowing tags in tag weight dashboard

### DISCUSSION THREAD

**Discussion by peteole** [forem/forem#17335] [Feature Requests]
Using forem, I find it pretty unintuitive that there is no central point where I can follow and unfollow tags. I think the tag weight section would be a great place for this.

**Comment by amywtlin** [comment]
Hello @peteole - thanks for the request! We are working on a big project rethinking /dashboard, and this is an area we look to improve with this project. I am looping in our teammate working on the project and moving this to `Next Up`. We don't have an ETA for the project though, as it is still in exploration.

Looping in @ludwiczakpawel this is relating to the /dashboard project - I had the same thought of "can I also unfollow the tag here while I see it?" when I looked at the new design iteration of the tag weighting page.

**Comment by ludwiczakpawel** [comment]
Too early to share the details but we had an idea to let users:
- follow a tag - that's what we have today,
- boost a followed tag - equivalent of today's _increasing the tag weight_,
- ignore a tag - which translates to "i don't want to see posts in my feed with certain tag".

With that in mind: 
- _following_ could be performed in multiple places obviously (as it is today),
- _boosting_ would initially be available through Dashboard's "Following tags" view,
- _ignoring_ would initially be available on tag's page.

But as @amywtlin mentioned - we don't really have any estimates right now in terms of WHEN this will be shipped as it's a part of a much bigger project that we're still trying to investigate and evaluate.

**Comment by peteole** [comment]
That's cool to hear! I think the tag weights are cool, but a bit too fine. I have actually never entered values except 0 and 1 till now :)
